graphql/cxs-impl/src/main/java/org/apache/unomi/graphql/commands/ProcessEventsCommand.java

```diff
@@ -104,7 +104,7 @@ private void processStaticFields(
                         }
 
                     } catch (Exception e) {
-                        LOG.debug("Process field {} is failed", eventProcessor.getFieldName(), e);
+                        LOG.warn("Process field " + eventProcessor.getFieldName() + " is failed", e);
```

Comment:
Can you use the Log formatter instead of the string concat? 
Also, this can be split in two, warn with the failed field, and keep the stack trace log for the debug.